config/300-revision.yaml

```diff
@@ -28,3 +28,13 @@ spec:
     - knative
     - serving
   scope: Namespaced
+  additionalPrinterColumns:
+  - name: Serving State
```

Comment:
Can you make this match the suggestions for Route in #1828?

I think the 3 columns we want (for now) are ServiceName, Ready, and Reason